### DISCUSSION TITLE
Are there some kinds of logfile?

### DISCUSSION THREAD

**Discussion by oliuo1** [rustdesk/rustdesk#2929] [General]
first of all: great project!

1. Logfile Serverside: See when who made use of the server, direct connection or not, encrypted or not

Why? to know who is using a self hosted server

2. Logfile Clientside: See when a client was used to establish a connection, same data described above

**Comment by jkirk** [comment]
Hey... just had the same question and found them here: `$HOME/.local/share/logs/RustDesk`
